zproject/computed_settings.py

```diff
@@ -312,7 +312,12 @@ def get_dirs(self) -> List[PosixPath]:
         DATABASES["default"]["OPTIONS"]["sslmode"] = REMOTE_POSTGRES_SSLMODE
     else:
         DATABASES["default"]["OPTIONS"]["sslmode"] = "verify-full"
-
+elif get_config("postgresql", "database_user") != "zulip":
+    if get_secret("postgres_password") is not None:
+        DATABASES["default"].update(
+            PASSWORD=get_secret("postgres_password"),
+            HOST="localhost",
```

Comment:
Is `localhost` necessary here to ensure we don't use ident auth?

Reply:
Yes, this is needed to ensure we don't use `ident` auth.